scripts/rssFeedGen.py

```diff
@@ -0,0 +1,95 @@
+from bs4 import BeautifulSoup
+from datetime import datetime
+import time
+from email.Utils import formatdate
+import PyRSS2Gen
+import StringIO
+import re
+
+def extract_releases(html):
+    html_doc = open(html, 'r').read()
+    soup = BeautifulSoup(html_doc, "lxml")
+    divRelease = soup.find("div", {"id":"release-notes"})
+    releases = divRelease.findAll("div", {"id":re.compile("release-")})
```

Comment:
maybe use a class select `section` instead within the div containing all the releases